Formula/k/kea.rb

```diff
@@ -10,9 +10,15 @@ class Kea < Formula
 
   livecheck do
     url "ftp://ftp.isc.org/isc/kea/"
-    # Match the final component lazily to avoid matching versions like `1.9.10` as `9.10`.
-    regex(/v?(\d+\.\d*[02468](?:\.\d+)+?)$/i)
-    strategy :page_match
+    regex(/^v?(\d+\.\d*[02468](?:\.\d+)*)$/i)
+    strategy :page_match do |page, regex|
+      page.lines(chomp: true).filter_map do |line|
+        candidate = line.split.last
+        next false unless regex.match?(candidate)
+
+        candidate
+      end
+    end
```

Comment:
```suggestion
    regex(/\s+v?(\d+\.\d*[02468](?:\.\d+)*)$/i)
```

We can achieve something similar without needing a `strategy` block by adding a whitespace delimiter to the start of the regex. Since the regex is anchored to the end of each line, I don't think there's much benefit to the `strategy` block approach over the regex in this suggestion.

Reply:
The benefit is using the same regex as other formulae for better `grep`-ability.

If we don't care about that, then the existing regex already works.